cuda/private/toolchain.bzl

```diff
@@ -42,15 +42,6 @@ cuda_toolchain = rule(
 CPP_TOOLCHAIN_TYPE = "@bazel_tools//tools/cpp:toolchain_type"
 CUDA_TOOLCHAIN_TYPE = "//cuda:toolchain_type"
 
-# buildifier: disable=unused-variable
-def use_cpp_toolchain(mandatory = True):
```

Comment:
Please don't remove this instance, some old versions of bazel don't have it defined, so it was copied.  We better leave it alone before 5.x phasing out.